.github/workflows/dhis2-verify-app.yml

```diff
@@ -75,11 +63,6 @@ jobs:
             - uses: c-hive/gha-yarn-cache@v1
             - run: yarn install --frozen-lockfile
 
-            # Can be removed if translations aren't required for tests,
-            # or if not using the App Platform
-            - name: Generate translations
-              run: yarn d2-app-scripts i18n generate
```

Comment:
is this safe to remove, given that we're using translations in our tests? https://github.com/dhis2/line-listing-app/blob/4bc7bad81fbdd4ef0b3c2a7a0ab03228ff2d84fd/cypress/integration/translations.cy.js#L37

Reply:
Yes bc this is only for Jest tests.

The cypress-io/github-action does all the installing and building needed to run the e2e tests (as you can see they are passing)